api/python/provisioner/commands/__init__.py

```diff
@@ -48,7 +48,9 @@
     CONTROLLER_BOTH,
     SSL_CERTS_FILE,
     SEAGATE_USER_HOME_DIR, SEAGATE_USER_FILEROOT_DIR_TMPL,
-    REPO_CANDIDATE_NAME
+    REPO_CANDIDATE_NAME,
+    RELEASE_INFO,
+    ReleaseInfo
```

Comment:
There's repetition and also the case discipline isn't followed.
Any specific reason?

Reply:
I renamed `RELEASE_INFO` constant for file name to `RELEASE_INFO_FILE` to avoid confusing. For classes in Python according to PEP8 CamelCase is used. As `ReleaseInfo` is class-enumeration I use this camel case for it.